app/services/authentication.js

```diff
@@ -5,9 +5,15 @@ exports.generateToken = user => {
   const tokenPayload = {
     email: user.email,
     name: user.name,
-    lastName: user.lastName
+    lastName: user.lastName,
+    role: user.role
   };
   return jwt.encode(tokenPayload, secret);
 };
 
-exports.decodeToken = token => jwt.decode(token, secret);
+const formatToken = token => token.split(' ')[1];
```

Comment:
What happen if token is sent without `Bearer <token>` format ?

Reply:
The error is catched in authentication middleware, so it will return a 400 code error with message 'invalid token'

Reply:
I don't see any catch [here](https://github.com/wolox-training/cr-express-js/pull/10/files#diff-517bd411a7d78b1ac6c3543a5e02227fR19)

Reply:
Check this comment too, maybe decode two times is not necessary (because two middlewares call decode token)